charts/newrelic-logging/templates/daemonset-windows.yaml

```diff
@@ -0,0 +1,139 @@
+{{- if and (include "newrelic-logging.areValuesValid" $)  $.Values.enableWindows }}
+{{- range .Values.windowsOsList }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    kubernetes.io/os: windows
+{{ include "newrelic-logging.labels" $ | indent 4 }}
+  name: {{ template "newrelic-logging.fullname" $ }}-windows-{{ .version }}
+  annotations:
+  {{- if $.Values.daemonSet.annotations }}
+{{ toYaml $.Values.daemonSet.annotations | indent 4 }}
+  {{- end }}
+spec:
+  updateStrategy:
+    type: {{ $.Values.updateStrategy }}
+  selector:
+    matchLabels:
+      app: {{ template "newrelic-logging.name" $ }}
+      release: {{ $.Release.Name }}
+      kubernetes.io/os: windows
+  template:
+    metadata:
+    {{- if $.Values.podAnnotations }}
+      annotations:
+{{ toYaml $.Values.podAnnotations | indent 8}}
+    {{- end }}
+      labels:
+        app: {{ template "newrelic-logging.name" $ }}
+        release: {{ $.Release.Name }}
+        kubernetes.io/os: windows
+        app.kubernetes.io/name: {{ template "newrelic-logging.name" $ }}
+        {{- if $.Values.podLabels}}
+{{ toYaml $.Values.podLabels | indent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ template "newrelic-logging.serviceAccountName" $ }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      terminationGracePeriodSeconds: 10
+      {{- if $.Values.image.pullSecrets }}
+      imagePullSecrets:
+{{ toYaml $.Values.image.pullSecrets | indent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ template "newrelic-logging.name" $ }}
+          image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag | default $.Chart.AppVersion  }}-{{ .imageTagSuffix }}"
+          imagePullPolicy: "{{ $.Values.image.pullPolicy }}"
+          securityContext: {}
+          env:
+            - name: ENDPOINT
+              {{- if (include "newrelic.nrStaging" $) }}
```

Comment:
Do we need to expose staging? Maybe we can expose only US and EU

Reply:
We are also doing this in the Linux daemonset, we can handle this, if needed, in a separate PR.